azuredevops/resource_project.go

```diff
@@ -195,12 +202,21 @@ func resourceProjectUpdate(d *schema.ResourceData, m interface{}) error {
 }
 
 func updateProject(clients *config.AggregatedClient, project *core.TeamProject, timeoutSeconds time.Duration) error {
-	operationRef, err := clients.CoreClient.UpdateProject(
-		clients.Ctx,
-		core.UpdateProjectArgs{
-			ProjectUpdate: project,
-			ProjectId:     project.Id,
-		})
+	var operationRef *operations.OperationReference
+
+	// project updates may fail if there is activity going on in the project. A retry can be employed
+	// to gracefully handle errors encountered for updates, up until a timeout is reached
+	err := resource.Retry(projectBusyTimeoutDuration, func() *resource.RetryError {
```

Comment:
Not time unit set ,default timeout will be `360ns`
```suggestion
	err := resource.Retry(projectBusyTimeoutDuration * time.Minute, func() *resource.RetryError {
```